index.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <div class="heading">
+      <h1 id="title">Kennedy's Survey Form</h1>
+      <p id="description">
+        Thank you for taking the time to help us improve the platform
+      </p>
+      <div class="form-main">
+        <form id="survey-form">
+          <div class="form-main">
+            <label id="name-label">
+              Name:
+              <input
+                type="text"
+                id="name"
+                required
+                placeholder="Enter Name..."
+              />
+            </label>
+          </div>
+          <div class="form-main">
+            <label id="email-label">
+              Email:
+              <input
+                type="email"
+                id="email"
+                required
+                placeholder="Enter Email Address..."
+              />
+            </label>
+          </div>
+          <div class="form-main">
+            <label id="number-label">
+              Age:
+              <input
+                type="number"
+                min="0"
+                max="9"
+                id="number"
+                placeholder="Enter Your Age..."
+              />
+            </label>
+          </div>
+
+          <div class="form-main">
+            <p>Would you recommend freeCodeCamp to a friend?</p>
+
+            <div class="form-main">
+              <input type="radio" name="radio" value="1" id="radio1" />
+              <label for="radio1">Definitely</label>
+            </div>
+
+            <div class="form-main">
+              <input type="radio" name="radio" value="2" id="radio2" />
+              <label for="radio2">Maybe</label>
+            </div>
+
+            <div class="form-main">
+              <input type="radio" name="radio" value="3" id="radio3" />
+              <label for="radio3">Not Sure</label>
+            </div>
+          </div>
+          <label style="width: 100%"
+            >Which option best describes your current role?</label
+          >
+
+          <select name="" id="dropdown">
+            <option value="0">Select an Option</option>
+
+            <option value="1">Challenges</option>
+
+            <option value="2">Projects</option>
+
+            <option value="3">Community</option>
+
+            <option value="4">Open Source</option>
+          </select>
```

Comment:
Please I suggest you do not add blank lines or spaces without a reason.

Reply:
> Please I suggest you do not add blank lines or spaces without a reason.

[OPTIONAL] This is a long tag, it would seem boring. It would be nice to pick by piece.

Reply:
@kennedyowusu I tagged the wrong one. I wanted to tag the one that is long. This comment is for it.

Reply:
Okay. thank you @Olamarx